src/pages/Internship.jsx

```diff
@@ -1,180 +1,152 @@
-{/* w-1440 3/4 */}
-{/*  */}
-
+import Navbar from "../components/Navbar"
 import React from "react";
 import Footer from "../components/Footer";
-import Instagram from '../images/Instagram.png'; 
-import fam_placeholder from '../images/fam_placeholder.png'
-import TotoPic from '../images/TotoPic.png'
-
+import HeaderBG from "../assets/InternHeaderBG.jpg"
+import Gear from "../assets/FullGear.png"
+import SunGear from "../assets/SunGear.png"
+import Sun from "../assets/FullSun.png"
+import Divider from "../assets/InternDutyDivider.png"
+import Testimonial from "../components/Testimonial";
+import CayzPhoto from "../assets/CayzIntern.jpg"
+import JhaneenPhoto from "../assets/JhaneenIntern.jpg"
+import ArlaghnPhoto from "../assets/ArlaghnIntern.jpg"
+import InternPhoto1 from "../assets/InternPhoto1.png"
+import InternPhoto2 from "../assets/InternPhoto2.png"
+import InternAppBg from "../assets/InternAppBG.png"
+import GrayDiamond from "../assets/GrayDiamond.png"
+import CornerTriangles from "../assets/CornerTriangles.png"
+import Dots1 from "../assets/Dots1.png"
+import Dots2 from "../assets/Dots2.png"
+import WhyInternBG from "../assets/WhyInternBG.png"
+import WhyInternRectangle1 from "../assets/WhyInternRectangle1.png"
+import WhyInternRectangle2 from "../assets/WhyInternRectangle2.png"
+import WhyInternRectangle3 from "../assets/WhyInternRectangle3.png"
 
 const Internship = () => {
     return (
-        <div className="flex flex-col"> {/* parent container */}
-            <header className="bg-[#cbc7b7] h-[840px] w-full flex flex-col items-center justify-center">  
-                <div className="text-center">
-                    <h1 className="text-black text-[128px] font-bold tracking-[.25em]">PROGRAMS</h1>
-                    <h2 className="text-black text-[40px] mt-[48px] tracking-[.25em]">INTERNSHIP</h2>
+        <>
+            <Navbar title="INTERNSHIP" subtitle="PROGRAMS" bg={HeaderBG}/>
+            <div className="my-40 mx-72 space-y-12 justify-center text-center">
+                <div className="font-semibold text-5xl">
+                    <span className="text-[#FFA523] ">Intern</span> with FUSION!
                 </div>
```

Comment:
The text section that follows the header needs to have the white background be specified